source/_integrations/alexa.smart_home.markdown

```diff
@@ -217,7 +217,9 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
   - Click the Smart Home skill you just created.
   - Click `Enable to use`.
   - A new window will open to direct you to your Home Assistant's login screen.
-    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
+    
+    note: If you have [IP Ban](https://www.home-assistant.io/integrations/http/#ip_ban_enabled) activated, after many try, there is the possibility receive a "403 Forbidden" return code from the Home assistant's login screen.
```

Comment:
So, how would many tries happen?

Some additional notes:

- Start sentences with capital letters.
- Don't use full URLs, instead, make URLs from the website root.